backend/generator.js

```diff
@@ -3,6 +3,10 @@ var exports = module.exports = {};
 var uuidV4 = require("uuid/v4");
 
 exports.executeScript = function (socket, formData) {
+  if (!isValid(formData)) {
```

Comment:
there is a difference between them, the backend script doesn't return a message as in the front end. Handling flash messages from the backend seemed strenous